src/parser/mod.rs

```diff
@@ -1323,13 +1324,24 @@ impl<'a> Parser<'a> {
                 expr: Box::new(expr),
                 trim_where,
                 trim_what: Some(trim_what),
+                trim_characters: None,
+            })
+        } else if self.consume_token(&Token::Comma) && dialect_of!(self is SnowflakeDialect) {
```

Comment:
`no FROM` (by doc Snowflake doesn't support `FROM` syntax) **and** `Snowflake` **and next is** `Comma`  -> `parse characters`